packages/core/src/anchor/ethereum/remote-cas.ts

```diff
@@ -78,12 +75,27 @@ export class RemoteCAS implements CASClient {
     return response.supportedChains as Array<string>
   }
 
+  /**
+   * Create an anchor request on CAS through `fetch`. If `waitForConfirmation` is `true`,
+   * it only returns after getting a confirmation from CAS. If `waitForConfirmation` is `false`,
+   * the method is a no-op, as it returns a stub Pending response. Here we assume that some other component
+   * has already saved an anchor request to AnchorRequestStore.
+   */
   async create(
     carFileReader: AnchorRequestCarFileReader,
     waitForConfirmation: boolean
   ): Promise<AnchorEvent> {
-    const response = await firstValueFrom(this.create$(carFileReader, waitForConfirmation))
-    return parseResponse(carFileReader.streamId, carFileReader.tip, response)
+    if (waitForConfirmation) {
+      const response = await firstValueFrom(this.create$(carFileReader, waitForConfirmation))
```

Comment:
I'm a little confused about the relationship between `waitForConfirmation` and the `shouldRetry` arg to `create$`.  It looks like in the code that exists in develop today, `shouldRetry` is set to the *opposite* of `waitForConfirmation`, whereas in this PR its set to the same thing.  But I actually don't really understand why `waitForConfirmation` should have any affect on the retry behavior?

Reply:
Maybe now that we have retries by way of the anchor processing loop, there's no need for retries at this layer at all?  Or maybe a fixed number of retries here, one or two retries, and then stop, instead of retrying indefinitely as seems to be the behavior currently.